lit-next.code-workspace

```diff
@@ -29,33 +29,57 @@
       "path": "packages/lit-starter-js"
     },
     {
-      "name": "localize",
-      "path": "packages/localize"
+      "name": "labs/ssr",
```

Comment:
Does it really help to have "labs" in the name? Could we make these changes separately?

Reply:
I figure I'm the only one using this... 😛

It helps because afaict you have to have the package listed here for its launch config to show up in the UI (otherwise there's not a ton of value listing each package as a workspace folder, you can just open the monorepo folder and browse the file tree). But if they're gonna be listed, I find not having them roughly match the file tree disorienting.

Maybe this argues we should just have one big launch config in the root and not per package. 🤷 

I can move it out of the PR if you want to debate it I guess.

Reply:
Oh, I use it. It used to at least make TypeScript behave a lot better because VS Code could understand what was a project and how to resolve references. tsconfig project references may have made that better, but we've removed those recently.

I guess one approach is to have a checked-in base workspace file and if people want to customize they can make their own. We could add an entry to .gitignore to put your personal fork in a known location.